example/example.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html ng-app="demoApp">
+  <body>
+    <main>
+      <ngc-omnibox></ngc-omnibox>
+    </main>
+
+    <script src="../dist/ngc-omnibox.js"></script>
+    <script type="text/javascript">
+      var demoApp = angular.module('demoApp', ['ngc.omnibox']);
```

Comment:
There's no need to include `var demoApp =` it seems like. Let's not encourage polluting the global scope :)